### PR TITLE
CO-562 hide passive owner claims from retry and failed status

### PR DESCRIPTION
**Pull request by Kbediako**
## What
- Reconcile passive released Backlog provider-owner failed run summaries out of current selected/discovered status projection.
- Add CO-558-shaped regressions for selected status, discovered status, real failed active-run preservation, and terminal retry exclusion.
- Record docs-freshness owner guidance that merged `docs:freshness:maintain` owners should remain `Backlog` while retained cohorts remain visible.

## Why
CO-562 covers a stale `failed` run summary that kept surfacing a released `Backlog` docs-freshness owner as current failed work after provider-intake had released it as `provider_issue_released:not_active` with `retry_queued=null`.

## Validation
- `node scripts/delegation-guard.mjs`
- `node scripts/spec-guard.mjs --dry-run`
- `npm run build`
- `npm run lint` (0 errors; existing `DelegationMcpHealth.test.ts` `any` warnings)
- `npm run test`
- `npm run docs:check`
- `npm run docs:freshness`
- `npm run repo:stewardship`
- `node scripts/diff-budget.mjs`
- `npm run pack:smoke`
- Manifest-backed standalone review: `review_verdict=clean`, enforce contract valid/clean.

Linear: CO-562

## Summary by CodeRabbit

* **Bug Fixes**
  * Passive released-owner claims no longer appear as active retrying/failed due to stale run summaries; reconciliation now treats them as cancelled/display-passive.

* **Behavioural Changes**
  * Retry-related fields are preserved when absent from intake data to avoid misclassification of claim state.

* **Tests**
  * New and expanded tests for selected-run reconciliation, compatibility projection and released-owner edge cases.

* **Documentation**
  * Added PRD, tech spec, action plan, task checklists and registry entries with acceptance criteria and rollout guidance.

[Review Change Stack](https://app.coderabbit.ai/change-stack/Kbediako/CO/pull/840?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: orchestrator/src/cli/control/selectedRunProjection.ts [hunk 1]
```diff
@@ -2101,6 +2108,28 @@ function isActiveLookingProviderLinearWorkerManifestStatus(status: string): bool
...
+    return false;
+  }
+  const workflowState = classifyProviderLinearWorkflowState({
+    state: claim.issue_state,
+    state_type: claim.issue_state_type
+  });
+  return (
+    workflowState.normalizedState === 'backlog' ||
+    workflowState.normalizedStateType === 'backlog'
+  );
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts:2150]: @coderabbitai Fixed in commit 73362b7929: the passive-release predicate now requires explicit empty retry metadata (`retry_queued`, `retry_attempt`, `retry_due_at`, and `retry_error` all null), and `tests/selected-run-projection.spec.ts` now covers the missing retry metadata case so it remains visible as failed. Validation rerun: `npm run build`, `npm run lint`, `npm run test`.

File: orchestrator/src/cli/control/selectedRunProjection.ts [hunk 2]
```diff
@@ -2101,6 +2108,31 @@ function isActiveLookingProviderLinearWorkerManifestStatus(status: string): bool
...
+function isPassiveReleasedProviderLinearWorkerOwnerFailedRun(
+  context: Pick<ControlCompatibilitySourceContext, 'rawStatus'>,
+  claim: ProviderIntakeClaimRecord
+): boolean {
+  if (
+    context.rawStatus !== 'failed' ||
+    claim.state !== 'released' ||
+    claim.reason !== 'provider_issue_released:not_active' ||
+    claim.retry_queued !== null ||
+    claim.retry_attempt !== null ||
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts]: Fixed in commit 3970c52737: historical retry-attempt counters are treated as idle when the release is `provider_issue_released:not_active`, Backlog, same-run, and has no queued retry due/error. The retry-state presenter also suppresses that passive released owner retry metadata from current status.

File: orchestrator/src/cli/control/selectedRunProjection.ts [hunk 3]
```diff
@@ -2101,6 +2108,31 @@ function isActiveLookingProviderLinearWorkerManifestStatus(status: string): bool
...
 
+function isPassiveReleasedProviderLinearWorkerOwnerFailedRun(
+  context: Pick<ControlCompatibilitySourceContext, 'rawStatus'>,
+  claim: ProviderIntakeClaimRecord
+): boolean {
+  if (
+    context.rawStatus !== 'failed' ||
+    claim.state !== 'released' ||
+    claim.reason !== 'provider_issue_released:not_active' ||
+    claim.retry_queued !== null ||
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts]: Fixed in commit 3970c52737: passive released Backlog owner rows now accept explicit idle retry metadata (`retry_queued` null or false, no due/error, finite/null historical attempt), so `retry_queued:false` no longer keeps a passive owner failed summary visible. Focused projection tests and full `npm run test` passed.

File: orchestrator/src/cli/control/selectedRunProjection.ts [hunk 4]
```diff
@@ -1931,13 +1935,16 @@ function resolveProviderLinearWorkerRunArtifactReconciliation(
   allContexts: ControlCompatibilitySourceContext[],
   providerIntakeState: ProviderIntakeState
 ): ProviderLinearWorkerRunArtifactReconciliationRecord | null {
+  const claim = findProviderLinearWorkerClaimForContext(providerIntakeState, context);
+  const passiveReleasedOwnerFailedRun =
+    claim !== null && isPassiveReleasedProviderLinearWorkerOwnerFailedRun(context, claim);
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts:1941]: Fixed in commit 3970c52737: passive owner reconciliation now requires the released claim run identity to match the failed run artifact before it can cancel the failed summary. Added a regression that a later passive same-issue release for a different run leaves the real failed run visible.

File: orchestrator/src/cli/control/selectedRunProjection.ts [hunk 5]
```diff
@@ -1931,13 +1935,16 @@ function resolveProviderLinearWorkerRunArtifactReconciliation(
...
   if (
     !isProviderLinearWorkerReconciliationSource(context) ||
-    !isActiveLookingProviderLinearWorkerManifestStatus(context.rawStatus)
+    (!isActiveLookingProviderLinearWorkerManifestStatus(context.rawStatus) &&
+      !passiveReleasedOwnerFailedRun)
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts:1945]: Fixed in c8ffe6afa0: reconciled non-failed provider-worker artifacts now clear stale lastError instead of preserving the old failed-run summary. Added CO-558 regression assertions for selected passive-owner and collection projection payloads. Validation rerun: npm run test -- tests/selected-run-projection.spec.ts, npm run build, npm run lint, npm run test, node scripts/spec-guard.mjs --dry-run, git diff --check, npm run docs:check, npm run docs:freshness, npm run repo:stewardship, node scripts/diff-budget.mjs, npm run pack:smoke.

File: orchestrator/src/cli/control/selectedRunProjection.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/selectedRunProjection.ts:1941]: Fixed in e50a1025e8 by including passive released owner claims in scoped compatibility discovery and adding a >16-claim provider-intake regression for the CO-558 shape. Focused projection test now passes: 14/14.
